pkg/handler/storage/client.go

```diff
@@ -66,9 +66,15 @@ func convertV2(in *regionv1.FileStorage) *openapi.StorageV2Read {
 	return &openapi.StorageV2Read{
 		Metadata: conversion.ProjectScopedResourceReadMetadata(in, in.Spec.Tags),
 		Spec: openapi.StorageV2Spec{
-			Attachments: &openapi.StorageAttachmentV2Spec{},
-			StorageType: openapi.StorageTypeV2Spec{},
-			Size:        in.Spec.Size.String(),
+			Attachments: &openapi.StorageAttachmentV2Spec{
+				NetworkIDs: convertAttachmentsList(in.Spec.Attachments),
+			},
+			StorageType: openapi.StorageTypeV2Spec{
+				NFS: &openapi.NFSV2Spec{
+					RootSquash: in.Spec.NFS.RootSquash,
```

Comment:
`in.Spec.NFS` is a pointer which may be `nil`, and when it is `nil` this will panic.